### PR TITLE
Fix wrong reference name in ClairSecurityScannerAPI

### PR DESCRIPTION
**Pull request by kleesc**
### Description of Changes

* Fix a few references using the wrong name

#### Issue: https://issues.redhat.com/browse/PROJQUAY-620

**TESTING** ->

**BREAKING CHANGE** ->

---

## Reviewer Checklist

- [ ] It works!
- [ ] Comments provide sufficient explanations for the next contributor
- [ ] Tests cover changes and corner cases
- [ ] Follows Quay syntax patterns and format

### CODE REVIEW

File: data/secscan_model/test/test_secscan_interface.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def test_load_security_information(repository, v4_whitelist, initialized_db):
     "next_token, expected_next_token",
     [
         (None, SplitScanToken("v4", V4ScanToken(56))),
-        (SplitScanToken("v4", V4ScanToken(1)), SplitScanToken("v4", None)),
```
Comment (alecmerdler) [inline data/secscan_model/test/test_secscan_interface.py:50]: We still need to test this case.

Comment (kleesc) [reply data/secscan_model/test/test_secscan_interface.py:50]: Is it a valid case though? Looking at what `perform_indexing` does, it looks like the it will always return a token if there's a `Manifest` with a higher `id` than the one parameterized for that test.

We can see from the previous one that there is one with `id=56`...

Comment (kleesc) [reply data/secscan_model/test/test_secscan_interface.py:50]: And fallback to `v2` if there is not.
